common/src/thrift/file_system_master.thrift

```diff
@@ -188,6 +188,30 @@ service FileSystemMasterClientService extends common.TachyonService {
    */
   bool unmount( /** the path of the tachyon mount point */ 1: string tachyonPath)
     throws (1: exception.TachyonTException e, 2: exception.ThriftIOException ioe)
+
+  /**
```

Comment:
similar to the protobuf logic, we could just have a single setAcl Thrift endpoint and an options struct that identifies which fields to set